hw_review/homework1.md

```diff
@@ -1,45 +1,336 @@
-## Homework1 review
-
-### Homework1 evaluation
-- Maximum score is +1 minimum is 0.
-- PR with 120+ rows evaluates as +1.
-- PR with 80+ rows evaluates as +0,5.
-- PR that doesn't pass any of PR checklist checks takes -0,5.
-- PR that doesn't pass any 2 of PR checklist checks takes -1.
-- All review comments should be addressed. Missed comments causes score 
-penalties.
-
-### PR checklist
-Please make sure you've passed PR checklist
-- [ ] my PR passes CI checks (flake8 and pycodestyle)
-- [ ] my PR named according guidelines: `Homework1: Name Surname`
-- [ ] my PR contains complete tasks descriptions for all tasks
-- [ ] my PR made from fork feature brunch to maxim master branch.
-- [ ] my PR don't contain only code that solves Homework1 tasks.
-Redundant files (configs, etc. ) are put to .gitignore
-
-
-### Review notes:
-1. It looks much more clear if you task description stands closely to task and 
-separated from other tasks with two blank lines: 
-    ```python
-   pass  # last line of previous task
+# 1 line: Output
+
+print ('Hello, world')
+
+#2 lines: Input, assignment
```

Comment:
It looks much more clear if your task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of the previous task
    
 
   # next task docstring
   pass  # my next task first line  
```